crates/db_schema/src/impls/instance.rs

```diff
@@ -37,15 +34,13 @@ use lemmy_db_schema_file::schema::{
   federation_queue_state,
   instance,
   instance_actions,
-  local_site,
-  site,
 };
 use lemmy_utils::error::{LemmyErrorExt, LemmyErrorType, LemmyResult};
 
 impl Instance {
   /// Attempt to read Instance column for the given domain. If it doesn't exist, insert a new one.
   /// There is no need for update as the domain of an existing instance cant change.
-  pub async fn read_or_create(pool: &mut DbPool<'_>, domain_: String) -> LemmyResult<Self> {
+  pub async fn read_or_create(pool: &mut DbPool<'_>, domain_: &str) -> LemmyResult<Self> {
```

Comment:
Changed param to reference to avoid unnecessary clones and make test code cleaner.